lib/errors.js

```diff
@@ -24,6 +24,7 @@ class CliError extends Error {
 
         this.context = options.context || {};
         this.options = options;
+        this.help = 'Please refer to https://docs.ghost.org for troubleshooting.'
```

Comment:
Sure! Wasn't sure which link you'd prefer, so was kinda expecting feedback here ;)